modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/v5_1_5/test/DDMStructureLayoutUpgradeProcessTest.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.dynamic.data.mapping.upgrade.v5_1_5.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.dynamic.data.mapping.upgrade.BaseDDMStructureLayoutUpgradeProcessTestCase;
+
+import org.junit.runner.RunWith;
+
+/**
+ * @author Paulo Albuquerque
+ */
+@RunWith(Arquillian.class)
+public class DDMStructureLayoutUpgradeProcessTest
+	extends BaseDDMStructureLayoutUpgradeProcessTestCase {
+
+	@Override
+	protected String getPaginationMode() {
+		return "paginated";
```

Comment:
@p-albuquerque don't we have a constant for this?

Reply:
@carolmariaabb  No, we don't. (at least I didn't find anywhere)

The upgrade itself use this way:
https://github.com/liferay/liferay-portal/blob/244bbc192a146e052d210a67eaf1f19d01865273/modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_1_5/DDMStructureLayoutUpgradeProcess.java#L60